omf/fileio/geoh5.py

```diff
@@ -1,26 +1,48 @@
+from __future__ import annotations
 
+import warnings
 from abc import ABC, abstractmethod
-
-from numpy import ndarray, c_, r_
-
-from omf.base import UidModel
-from omf.data import ScalarData, ScalarArray
-from omf.pointset import PointSetElement, PointSetGeometry
 from contextlib import contextmanager
-from geoh5py.workspace import Workspace
+from pathlib import Path
+
+import numpy as np
 from geoh5py.data import Data
-from geoh5py.objects import Points
+from geoh5py.groups import RootGroup
+from geoh5py.objects import BlockModel, Curve, Grid2D, Points, Surface
 from geoh5py.shared import Entity
-from pathlib import Path
+from geoh5py.workspace import Workspace
+
+from omf.base import Project, UidModel
+from omf.data import ScalarArray, ScalarData
+from omf.lineset import LineSetElement, LineSetGeometry
+from omf.pointset import PointSetElement, PointSetGeometry
+from omf.surface import SurfaceElement, SurfaceGeometry, SurfaceGridGeometry
+from omf.volume import VolumeElement, VolumeGridGeometry
 
 
-class GeoH5Writer(object):
+class OMFtoGeoh5Implementation(Exception):
+    """Custom error message for rotation not implemented by geoh5."""
```

Comment:
not necessarily just because of rotation. I suppose some type might not be supported

Reply:
Done